components/tikv_util/src/lib.rs

```diff
@@ -49,6 +49,7 @@ pub mod logger;
 pub mod metrics;
 pub mod mpsc;
 pub mod security;
+pub mod string;
```

Comment:
If it's only used in `tidb_query`, I suggest placing it in `tidb_query_shared_expr` :)